tox.ini

```diff
@@ -4,15 +4,16 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py35-django22,py38-django{22,30},quality
+envlist = python{3.5}-django22,python{3.8}-django{22,30,31},quality
 
 [testenv]
-passenv = CI TRAVIS TRAVIS_*
```

Comment:
Don't we still need this until we actually remove `.travis.yml`?

Reply:
reverted the change for now.